### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by haraphat01**
I had created this pull request earlier, but I had a conflict with the development branch, I had to merge my last branch to the development branch before I was able to resolve the issue.

Here is the final code for the calculator testing, which contains the test for Index, Home, Quote, and Calculate components.

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @haraphat01 :wave:

 __You've made laudable progress in this milestone.__
- **Your test cases are running and passing successfully :100:** 
- **All linter checks are running without errors :+1:** 

__However, some changes are required before this milestone can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] Kindly include instructions on how to use your app (install dependencies, running your project locally) and how to run your test cases in your readme file.

**[ OPTIONAL CHANGES ]**
- [ ] I know that you've added test cases for Calculate.js, which in turn affects the Operate.js file. However, I think you should add separate test cases for the Operate.js script. This is because. The milestone is based on unit testing and should be therefore be treated as such.


* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @haraphat01 :hand: 
Great job implementing the requested changes
However, we still have some changes to make

- [ ] Right now following the README instructions to run the app does not work. Running `npm run start` throws an error related to the missing index.html file inside the `public` directory.

Happy coding :computer: 
**Sérgio Torres**

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**
